src/content/en/2020/markup.md

```diff
@@ -457,22 +483,58 @@ Yet the most popular attribute is `class`—with almost 3 billion occurrences in
 | `rel` | 171,802,460 | 1.96% |
 | `value` | 140,666,779 | 1.61% |
 
-<figcaption>{{ figure_link(caption="Top 10 attributes.", sheets_gid="1348855449", sql_file="pages_almanac_by_device_and_attribute_name_frequency.sql") }}</figcaption>
+<figcaption>{{ figure_link(caption="Top 10 attributes by frequency of use.", sheets_gid="1348855449", sql_file="pages_almanac_by_device_and_attribute_name_frequency.sql") }}</figcaption>
 </figure>
 
-The `value` attribute, which specifies the value of an `input` element, surprisingly completes the top ten—surprising because on our end, subjectively, we didn’t get the impression values were used that frequently.
+The most popular attribute is `class`, with nearly 3 billion occurrences in our dataset and constituting 34% of all attributes in use. `class` is by far the most prevalent attribute.
+
+The `value` attribute, which specifies the value of an `input` element, surprisingly completes the top ten. It's surprising to us because, subjectively, we didn't get the impression values were used that frequently.
```

Comment:
Please markup “values” as code, but like “`value`s” (I think we need to for clarity, as values without code markup would seem to apply to any type of value).